rhel/vex/parser.go

```diff
@@ -331,9 +352,59 @@ func (c *creator) lookupVulnerability(vulnKey string, protoVulnFunc func() *clai
 	return &c.fixedVulns[idx], ok
 }
 
+type ranger struct {
+	lowest map[string]*claircore.Range
+}
+
+func newRanger() *ranger {
+	return &ranger{
+		lowest: map[string]*claircore.Range{},
+	}
+}
+
+func (r *ranger) add(packageName, fixedInVersion string) (*claircore.Range, error) {
+	var rng = &claircore.Range{}
+
+	if fixedInVersion == "" {
+		zeroVer := &rhctag.Version{}
+		rng.Lower = zeroVer.Version(true)
+		highestVer := rhctag.Version{
+			Major: math.MaxInt32, // Everything is vulnerable
+		}
+		rng.Upper = highestVer.Version(true)
```

Comment:
should this be `false` or does it really not matter much?

Reply:
I think `true` is correct as the major is already maxed out, I think it's better as `v2147483647.0.0` rather than `v2147483647.0.2147483647`